### PR TITLE
22-1995 form submit tracking updates

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description

Updated GA tracking for 22-1995 submit. The new fields are behind a prod flag because we don't have a method to check the feature flag on submit.  This will be removed when the 10203 goes live.

[ZenHub Issue #11700](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/11700)

## Testing done
Tested locally and QA reviewed

## Screenshots
N/A

## Acceptance criteria
- [x] Tracked fields match story list

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/1995/helpers.jsx [hunk 1]
```diff
@@ -27,3 +28,63 @@ export const display1995StemFlow = form =>
...
+    }
+    return value ? 'Yes' : 'No';
+  };
+
+  return {
+    'benefits-used-recently': formData.benefit,
+    'new-service-periods-to-record': yesNoOrUndefined(
+      formData['view:newService'],
+    ),
+    'service-details': _.get(formData, 'toursOfDuty', []).map(tour => ({
```
Comment (bkjohnson) [inline src/applications/edu-benefits/1995/helpers.jsx:63]: You don't need to import lodash for this.  Try this instead:

```js
     'service-details':  (formData.toursOfDuty || []).map(tour => ({
```

Comment (dmccurdybah) [reply src/applications/edu-benefits/1995/helpers.jsx]: Thanks!

**Review by bkjohnson — APPROVED** [review]
I'm not sure why this PR still needs an approval since it got the bah one, but it looks :ok_hand:

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**